camayoc/tests/qcs/api/v1/credentials/test_manager_creds.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+"""Tests for the ``Credential`` API endpoint.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+import requests
+
+import pytest
+
+from camayoc.constants import (
+    QCS_HOST_MANAGER_TYPES,
+    QCS_BECOME_METHODS,
+)
+from camayoc.qcs_models import Credential
+from camayoc.utils import uuid4
+
+
+@pytest.mark.parametrize('method', QCS_BECOME_METHODS)
+@pytest.mark.parametrize('cred_type', QCS_HOST_MANAGER_TYPES)
+def test_negative_create_with_become(
+        cred_type, shared_client, cleanup, method):
+    """Create a credential with username and password.
+
+    :id: d04e3e1b-c7f1-4cc2-a4a4-a3d3317f95ce
+    :description: Create a credential with a user name and password
```

Comment:
I can't tell from the docstring what is the real purpose of this test. As I see by the code is that you are trying to create a credential not only with username and password but also become information and that should not be accepted by the HOST_MANAGER_TYPES. Can you please update it?